ansible/README.local.md

```diff
@@ -1,4 +1,24 @@
 # How to configure Jukebox to run on your local computer or laptop 
-1. Run `ansible-playbook build-local.yml`, and wait for it to finish installing.
-2. Go to https://localhost in your web browser, and accept the self-signed
-certificate, then try out the site.
+
+1. Setup your google and ego secrets
+    - GoogleOauth secrets in "~/secrets/oauth_secrets.yml"
+    - Ego secrets in "~/secrets/ego.yml"
+2. Make sure Ego is setup with the email to be used with google authentication and it has desired scope.
+   - The desired scope for jupiterhub is set in ego.yml
```

Comment:
Jupyterhub is spelled with a 'y'; don't ask me why.

Reply:
Haha